generators/python/tests/sdk/test_root_client_generator.py

```diff
@@ -52,28 +54,20 @@ def test_generated_root_client_builder() -> None:
 
     snippet = SourceFileFactory(should_format=True).create_snippet()
     snippet.add_expression(generated_root_client.sync_instantiation)
-    assert (
-        snippet.to_str()
-        == """from acme.client import Acme
-from acme.environments import AcmeEnvironments
-
-client = Acme(
-    base_url="acme.io",
-    environment=AcmeEnvironments.PRODUCTION,
-)
-"""
+    assert snippet.to_str() == textwrap.dedent(
+        """\
+        from acme.client import Acme
+        from acme.environments import AcmeEnvironments
+        client = Acme(base_url="acme.io", environment=AcmeEnvironments.PRODUCTION, )
```

Comment:
Hmm.. this looks off - why is there a `,` in the same line?

Reply:
it’s how we generate the code. it’s different because previously we were hard coding the string during code generation, but now we’re leveraging the AST nodes.

Ruff figures out how to format it afterwards.

Reply:
Understood, thanks for that!